app/src/main/res/layout/dashboard_list_item_category.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <variable name="category" type="String" />
+        <variable name="isSelected" type="Boolean" />
+        <variable name="callbacks" type="org.cru.godtools.ui.dashboard.tools.CategoryAdapterCallbacks"/>
+    </data>
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <androidx.cardview.widget.CardView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="4dp"
+            android:minHeight="64dp"
+            android:onClick="@{() -> callbacks.onCategorySelected(category)}"
+            android:padding="6dp"
+            app:cardBackgroundColor="@{isSelected ? @color/gray_80 : @color/white }"
+            app:cardCornerRadius="@dimen/card_corner_radius"
+            app:cardElevation="4dp"
+            tools:cardBackgrounColor="@color/gray_80">
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="20dp"
+                android:layout_marginTop="15dp"
+                android:layout_marginBottom="15dp"
+                android:text="@{category.replace('_', ' ')}"
```

Comment:
you should use the getToolCategory method to resolve the display name taking localization into account